src/services/cartService.js

```diff
@@ -0,0 +1,11 @@
+const database = require("typeorm");
+const cartDao = require("../models/cartDao");
+
+const cart = async (userId, productId, quantity) => {
+  const cart = await cartDao.cart(userId, productId, quantity);
+  return cart;
+};
```

Comment:
코드를 한 줄로 줄일 수 있을 것 같네요~
일반적으로 80 글자가 넘지 않는 경우에는 한 줄로 코드를 작성하는 것이 가독성에 좋습니다.

```suggestion
const cart = async (userId, productId, quantity) => await cartDao.getCarts(userId, productId, quantity);
```